posts/index.html

```diff
@@ -13,7 +13,7 @@ <h2>Blog</h2>
     <article>
       <header>
         <div class="date-orange">
-        <time datetime="2017-02-14">{{ post.date | date: site.date_format }}</time>
+        <time datetime="{{ post.date | date: '%Y-%m-%d' }}">{{ post.date | date: site.date_format }}</time>
```

Comment:
Thanks for this @barinali

Reply:
You are welcome. I realized it while getting influenced by the /posts page and thought I'd fix it while I'm at it. :)